api/cronTasks/weeklyReset.js

```diff
@@ -0,0 +1,32 @@
+const cron = require('node-cron');
+const cronJob = require('./cronModel');
+
+// RWD (Saturday-Monday)
+cron.schedule('* * * * * 6,0,1', () => {
+    console.log("It is working!!!");
```

Comment:
can we tailor these console logs to reflect the job being run?  i.e. "it is working: saturday-monday"

Reply:
I have updated